### ISSUE TITLE
API: include player IDs in "recent" games

### ISSUE THREAD

**Issue by lucasgautheron** [ScalaWilliam/ActionFPS#22]
Currently, the API doesn't include the player IDs (if relevant) into the player list associated to each game. If a player has been associated, his ID should appear in the list along with his name in  http://api.actionfps.com/recent/ as in http://api.actionfps.com/users/

The player field would become, for instance :
```json
"players":[
    {"id":"sanzo", "name":"w00p|Sanzo","score":362,"flags":1,"frags":38,"deaths":39},
    {"id":"master", "name":"CR7","score":470,"flags":1,"frags":34,"deaths":42},
    {"id":null, "name":"|BC?NarcotiK","score":336,"flags":0,"frags":32,"deaths":39}]}
]
```

Notice that in case no matching user was found, a special value is associated to the "id" field to make it clear (here i suggest null).

**Comment by ScalaWilliam** [comment]
Sure.

I'll do it as ```"user": "sanzo"```, and if there is none, then field isn't included.

**Comment by ScalaWilliam** [comment]
Closed by d5760091dfee4ca8595ec16062d0413ce3f33578
